### PR TITLE
Move domain to options

### PR DESCRIPTION
**Pull request by alexshtin**
For both `ClientOptions` and `WorkerOptions` domain is now optional with default value `default`.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,24 +5,38 @@ go 1.12
 require (
 	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
 	github.com/facebookgo/clock v0.0.0-20150410010913-600d898af40a
+	github.com/gogo/googleapis v1.3.2 // indirect
```
Comment (alexshtin) [inline go.mod]: Updated all dependencies.

File: internal/internal_worker.go [hunk 2]
```diff
@@ -122,6 +120,9 @@ type (
 
 	// workerExecutionParameters defines worker configure/execution options.
 	workerExecutionParameters struct {
+		// Domain name.
```
Comment (mfateev) [inline internal/internal_worker.go]: Add info about the default value

File: test/test_utils.go [hunk 3]
```diff
@@ -36,7 +34,7 @@ type Config struct {
 
 func newConfig() Config {
 	cfg := Config{
-		ServiceAddr: internal.LocalHostPort,
+		ServiceAddr: "localhost:7233",
```
Comment (mfateev) [inline test/test_utils.go]: I created a public constant for this.

File: internal/internal_worker.go (hunk 2 above)
Comment (alexshtin) [reply internal/internal_worker.go]: This is internal struct and it doesn't have default value. Public structs have default value in comment.

**Pull request merged**